inline-js/package.yaml

```diff
@@ -8,12 +8,8 @@ license: BSD3
 github: tweag/inline-js
 
 extra-source-files:
-  - CHANGELOG.md
   - LICENSE
-  - README.md
-
-data-files:
-  - jsbits/server.js
+  - package.yaml
```

Comment:
It looks a bit suspicious. Do we usually include `package.yaml` for `hpack`-based projects?